CONTRIBUTING.md

```diff
@@ -0,0 +1 @@
+If you would like to contribute, please see the [Contributing page](https://github.com/musescore/MuseScore/wiki/Contributing.md) on the GitHub Wiki. Thanks!
```

Comment:
as @shoogle mentioned earlier: in the Wiki the Contribution file lost the .md extension, so it needs to here in the URL too

Reply:
.md removed